include/crow/routing.h

```diff
@@ -1584,42 +1614,18 @@ namespace crow
             {
                 for (auto& per_method: per_methods_)
                 {
-                    if (std::get<0>(per_method.trie.find(req.url)))
+                    if (std::get<0>(per_method.trie.find(req.url))) //Route found, but in another method
                     {
-                        CROW_LOG_DEBUG << "Cannot match method " << req.url << " " << method_name(method_actual);
-                        res = response(405);
+                        std::string error_message(get_error(405, found, req, res));
```

Comment:
const? Also for 404.

Reply:
I didn't know `const` meant non mutable, I thought it meant "evaluated at compile time". The more you learn